client/blocks/login/social.scss

```diff
@@ -146,98 +62,6 @@
 			padding: 24px;
 		}
 	}
-
-	/**
-	* Crowdsignal Login
-	*/
-	@at-root .crowdsignal & {
-		.auth-form__social-buttons .auth-form__social-buttons-container .social-buttons__button {
-			display: none;
-
-			&.google {
-				display: block;
-				background-color: #fff;
-				margin-bottom: 20px;
-				box-shadow: none;
-				border: solid var(--color-neutral-10);
-				border-width: 1px 1px 2px;
-				border-radius: 2px;
-				box-sizing: border-box;
-				height: 45px;
-				padding: 13px 40px 12px;
-
-				span {
-					margin-left: 15px;
-					font-family: inherit;
-					font-style: inherit;
-					line-height: normal;
-					letter-spacing: normal;
-					font-weight: 600;
-					font-size: 0.875rem;
-				}
-
-				@include break-medium {
-					margin-bottom: 40px;
-				}
-			}
-		}
-	}
-
-	/**
-	* Jetpack Cloud
-	*/
-	@at-root .jetpack-cloud & {
-		max-width: 375px;
-		width: 100%;
-		padding: 16px;
-		margin-bottom: 0;
-
-		.auth-form__social-buttons .auth-form__social-buttons-container .social-buttons__button {
-			> svg {
-				margin-right: 0;
-			}
-
-			span {
-				font-weight: 400;
-				font-family: inherit;
-				letter-spacing: normal;
-				line-height: 22px;
-				margin-left: 9px;
-				margin-right: 0;
-			}
-		}
-
-		@include break-mobile {
-			padding: 24px;
-		}
-	}
-
-	/**
-	* WooCommerce Login
-	*/
-	@at-root .is-woo-passwordless & {
-		.auth-form__social-buttons .auth-form__social-buttons-container .social-buttons__button {
-			height: 48px;
-			/* stylelint-disable-next-line scales/radii */
-			border-radius: 8px;
-
-			> svg {
-				margin-right: auto;
-			}
-
-			span {
-				color: var(--studio-gray-100);
-				font-family: Inter, -apple-system, system-ui, sans-serif;
-				font-size: 1rem;
-				font-style: normal;
-				font-weight: 500;
-				line-height: 24px;
-				letter-spacing: 0.32px;
-				margin-left: -20px;
-				margin-right: auto;
-			}
-		}
-	}
 }
 
 .auth-form__social {
```

Comment:
All the sections in this block have been removed or modified because the styles have been centralized in `client/components/social-buttons/style.scss`